fe/fe-core/src/main/java/org/apache/doris/common/util/ProfileManager.java

```diff
@@ -132,8 +134,15 @@ public void pushProfile(RuntimeProfile profile) {
         if (profile == null) {
             return;
         }
-        
+
+        // filter profile that take more time than threshold
         ProfileElement element = createElement(profile);
```

Comment:
Currently, Doris will call this `pushProfile()` at 3 phases:
1. The beginning of the query.
2. During the query execution(report incremental)
3. The end of the query.

At phase 1 and 2, the `element.totalTimeMs` may be very small. so you may not be able to see the profile of current running queries. This is not a very big problem because it will be pushed successfully at phase 3, finally.
But can we change it like:

1. Allow pushing at phase 1 and 2, no matter what `element.totalTimeMs` is.
2. At phase 3, check the threshold and determine to push the final profile, or to delete it if the query time is too small.

Reply:
I understand that pushing profile does not need to care about the different phase, it will make the code look more difficult to understand.